File: Perl/Process.pl
#! /usr/bin/env perl

print <<"Process";
プロセスID: $$
Perl:       @{[$^X]}
バージョン: @{[$^V]}
OS:         @{[$^O]}

このコードの行番号: @{[__LINE__]}
パッケージ名:       @{[__PACKAGE__]}

では,3.14秒ほど居眠りします


Process

sleep(3.14);

File: PHP/Process.php
#! /usr/bin/env php
<?php

$i = function($v){return $v;};

print <<<"Process"
プロセスID: {$i(getmypid())}
PHP:        {$i(PHP_BINARY)}
バージョン: {$i(PHP_VERSION)}
OS:         {$i(PHP_OS)} ({$i(PHP_OS_FAMILY)})

このコードの行番号: {$i(__LINE__)}

では,3.14秒ほど居眠りします


Process;
sleep(3.14);

?>